docs/developer.md

```diff
@@ -163,6 +163,23 @@ Creating and using EKS clusters requires to you to have the [AWS IAM Authenticat
 go get github.com/kubernetes-sigs/aws-iam-authenticator/cmd/aws-iam-authenticator
 ```
 
+#### EKS ami image query script
+
+```
+K8S_VERSIONS=(
+  "1.11"
+  "1.12"
+  "1.13"
+  "1.14"
+)
+
+for version in ${K8S_VERSIONS[@]}; do
```

Comment:
In the past, this type of query returned also images that were not available for use yet. Our experience was that only those images are available for use that are published here: https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html, though thins may have changed since then.